User/InMemoryUserRepository.php

```diff
@@ -5,18 +5,21 @@
 /**
  * Class InMemoryUserRepository
  */
-class InMemoryUserRepository implements UserRepository
+class InMemoryUserRepository implements UserRepository, PasswordResetRepositoryInterface
 {
     /** @var array */
-    private $user = [];
+    private $users = [];
 
     public function __construct()
     {
-        $this->users[] = new User(
+        $user = new User(
             'wouter',
             'test',
             'Wouter Sioen'
         );
+        
+        $user->generatePasswordResetToken();
```

Comment:
waarom doet ge dat niet in de constructor van de user zelf? Het lijkt me niet zo logisch dat een repository dat moet doen?